docs/user/index.rst

```diff
@@ -90,14 +90,14 @@ to help you create fantastic documentation for your project.
 
 .. toctree::
    :maxdepth: 2
-   :hidden:
    :caption: How-to Guides
-   :glob:
+   :titlesonly:
```

Comment:
Using `:titlesonly:` is an experiment. I'll probably revert it so we can keep a consistent display across the documentation. But the FAQ item is quite a mess with all the FAQ entries in it.